integration/codegen.ts

```diff
@@ -31,7 +31,7 @@ async function generate(binFile: string, baseDir: string, parameter: string) {
   const options = optionsFromParameter(parameter || '');
   const typeMap = createTypeMap(request, options);
 
-  for (let file of request.protoFile) {
+  for (let file of request.fileToGenerate) {
```

Comment:
This causes compile errors due to the `generateFile` call on line 38 still wanting the `FileDescriptorProto` but `fileToGenerate` is just a `string[]`.

Reply:
I will take a look!